profilers-android/src/com/android/tools/idea/profilers/ProfileRunExecutor.java

```diff
@@ -95,6 +97,9 @@ public Icon getExecutorIcon(@NotNull Project project, @NotNull Executor executor
 
   @Override
   public boolean isApplicable(@NotNull Project project) {
-    return !ProjectFacetManager.getInstance(project).getFacets(AndroidFacet.ID).isEmpty();
+    ProjectFacetManager facetManager = ProjectFacetManager.getInstance(project);
+    return facetManager.hasFacets(AndroidFacet.getFacetType().getId()) ||
+           facetManager.hasFacets(ApkFacet.getFacetTypeId()) ||
```

Comment:
Are you sure any Gradle project can be profiled with Android profiler?

Reply:
Android Gradle projects, yes; they're the kind Android Studio normally uses. Did you look at GradleFacet?

Reply:
Ok, I see it now. GradleFacet name was a bit misleading.

Reply:
I agree! Thanks for taking a look.